### PR TITLE
docs(REVIEWING): add an overview of the PR process

### PR DESCRIPTION
**Pull request by ghost**
This is a rough draft (and long overdue), feel free to comment on it.

### CODE REVIEW

File: REVIEWING.md [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+credentials. The first one is a kind of "super admin" access key and secret
+key (see it as a login token and a password token), and the second kind is a
+user-generated access key and secret key.
+
+The first one is necessary at installation time, as it is used to configure
+Vault.
+
+The second one can only be generated and used once the product is up and
+running. The utilities to generate those credentials may also be useful for our
+users to setup a quick test platform.
```
Comment (mvaude) [inline REVIEWING.md]: Do we not speak about `migration` playbooks by purpose?

**Review by mvaude — CHANGES_REQUESTED** [review]
Thanks for this documentation!
I hope it will help to uniformize team members vision on what is expected from a PR and the related reviews.
I still have some questions regarding some parts of the processes defined.
Anyway, it is not huge request changes.

File: REVIEWING.md [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+
+  - Has it a marginal impact on other components? -> Integration tests
+
+  - Is it a big change for everyone? -> End to end tests
+
+  - Never forget to write regression tests to ensure that we'll never trigger
+  bugs again
+
+  - Never forget to write compatibility tests to avoid break our stored data
+  formats and protocols
```
Comment (michael-zapata) [inline REVIEWING.md]: Just because it was never done doesn't mean it shouldn't be.

File: REVIEWING.md [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+    be tolerated for a number of very small PRs quick to review and thus merge.
+
+  - Production of PRs and Review of PRs should observe a simple ratio. For each
+    PR produced and merged, you should be participating in the process of
+    merging two other PRs as a reviewer. This will allow to produce as many PRs
+    as we merge, and keep the number of open PRs stable
+
+- Documentation can go a long way for reviewers and knowledge transfer:
+
+  - JSDoc for an API brings a lot to the  table, and eases the review
```
Comment (michael-zapata) [inline REVIEWING.md]: This was written at a time when we only ever did JS, aside from a few python scripts that were handled by a default `pylint`, the strictest linter when it comes to python code.

Comment (mvaude) [inline REVIEWING.md]: I was thinking about a more generic way to express it (for example a `rust` project is coming soon ;)).
Then shouldn't we update it to specify each linter we are using? Or express it differently as I was explaining?

File: REVIEWING.md [hunk 4]
```diff
@@ -0,0 +1,274 @@
...
+    merging.
+
+## Reviewing your peers, the gentle way
+
+We're all human beings, and we can't possibly know everything about every
+project, asking questions is perfectly fine when some piece of code seems a bit
+obscure. Take care not to block the process for a trivial question, that could
+easily be ignored.
+
+Giving an approval review should be incidental, not the standard. This doesn't
```
Comment (mvaude) [inline REVIEWING.md]: I understand the meaning: it doesn't have to be a reflex to just approve some a PR. Define it as an `incident` is a bit too much in my opinion :)

File: REVIEWING.md (hunk 3 above)
Comment (michael-zapata) [inline REVIEWING.md]: Yeah, I don't disagree with you at all, this is just a very old PR that was written in a wildly different context (We didn't even have maintainers at that point in time). I'll try to update that when we feel like moving forward with this.

**Comment by ghost** [comment]
Closing, feel free to pick it back.

**Pull request closed without merging**